src/calculateRentalCost.js

```diff
@@ -28,7 +28,11 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else {
+    return days >= 7 ? (days * 40) - 50 : (days * 40) - 20;
```

Comment:
The current logic applies a $20 discount for rentals of 3 or more days, even if the rental is 7 or more days. According to the task description, if the rental is for 7 or more days, a $50 discount should be applied instead. Consider restructuring the condition to check for 7 or more days first, and then check for 3 or more days.